### ISSUE TITLE
Docker: Official image is based on EOL Ubuntu 20.10

### ISSUE THREAD

**Issue by orgads** [grafana/grafana#74999]
### What happened?

Ubuntu kinetic (20.10) is no longer maintained, and even running apt-get update fails on it.

I couldn't find the source Dockerfile for the official image, but it is based on 20.10.

### What did you expect to happen?

It should be upgraded to 22.04 ASAP.

### Did this work before?

Yes.

### How do we reproduce it?

1.
2.
3.

### Environment (with versions)?

Grafana: 10.1.1-ubuntu
OS: Docker

### Grafana platform?

Docker

**Comment by usmangt** [comment]
Hi @orgads,

Thanks for opening this issue.

Can you please point us to the documentation link that you are following to setup for this Docker image?

**Comment by orgads** [comment]
I just use the official image:

```
$ docker run --rm --entrypoint bash grafana/grafana:10.1.1-ubuntu -c 'cat /etc/os-release'
PRETTY_NAME="Ubuntu 22.10"
NAME="Ubuntu"
VERSION_ID="22.10"
VERSION="22.10 (Kinetic Kudu)"
VERSION_CODENAME=kinetic
ID=ubuntu
UBUNTU_CODENAME=kinetic
```

**Comment by kayhannay** [comment]
Same here, we build a Docker image with Grafana as base image:

`FROM grafana/grafana:latest-ubuntu

ENV DEBIAN_FRONTEND="noninteractive"

RUN apt-get update \
 && apt-get install -y --no-install-recommends \
        ca-certificates \
        apt-transport-https \
        apt-utils
...`

This does not work any more, we get the following error:
`#7 [2/8] RUN apt-get update  && apt-get install -y --no-install-recommends         ca-certificates         apt-transport-https         apt-utils
#7 0.463 Ign:1 http://archive.ubuntu.com/ubuntu kinetic InRelease
#7 0.464 Ign:2 http://security.ubuntu.com/ubuntu kinetic-security InRelease
#7 0.539 Ign:3 http://archive.ubuntu.com/ubuntu kinetic-updates InRelease
#7 0.544 Err:4 http://security.ubuntu.com/ubuntu kinetic-security Release
#7 0.544   404  Not Found [IP: 185.125.190.39 80]
#7 0.615 Ign:5 http://archive.ubuntu.com/ubuntu kinetic-backports InRelease
#7 0.690 Err:6 http://archive.ubuntu.com/ubuntu kinetic Release
#7 0.690   404  Not Found [IP: 185.125.190.39 80]
#7 0.766 Err:7 http://archive.ubuntu.com/ubuntu kinetic-updates Release
#7 0.766   404  Not Found [IP: 185.125.190.39 80]
#7 0.841 Err:8 http://archive.ubuntu.com/ubuntu kinetic-backports Release
#7 0.841   404  Not Found [IP: 185.125.190.39 80]
#7 0.845 Reading package lists...
#7 0.854 E: The repository 'http://security.ubuntu.com/ubuntu kinetic-security Release' does not have a Release file.
#7 0.854 E: The repository 'http://archive.ubuntu.com/ubuntu kinetic Release' does not have a Release file.
#7 0.854 E: The repository 'http://archive.ubuntu.com/ubuntu kinetic-updates Release' does not have a Release file.
#7 0.854 E: The repository 'http://archive.ubuntu.com/ubuntu kinetic-backports Release' does not have a Release file.
#7 ERROR: process "/bin/sh -c apt-get update  && apt-get install -y --no-install-recommends         ca-certificates         apt-transport-https         apt-utils" did not complete successfully: exit code: 100`

The grafana image  with tag latest-ubuntu is still based on Ubuntu 22.10 'Kinetic Kudu' which is not supported any more. So you probably have to re-build the grafana images based on Ubuntu with a LTS release or the latest release of Ubuntu.

**Comment by khushijain21** [comment]
hello, does this require contribution?

**Comment by orgads** [comment]
Already fixed in 10.1.2.
